routes/planReview.js

```diff
@@ -15,27 +15,71 @@ router.post("/api/planReview/addReviewer", (req, res) => {
   }
   plans
     .findById(plan_id)
-    .then((plan) => {
+    .then(async (plan) => {
       if (plan.reviewers.indexOf(reviewer_id) < 0) {
-        //add reviwer_id to the plan
-        plan.reviewers.push(reviewer_id);
-        //add plan_id to reviewer's doc
-        users
+        //pull up reviewer info
+        await users
```

Comment:
Do you need `await` if you use `then()`? just a question since I'm not familiar with JS

Reply:
No, in fact you don't.